build.sbt

```diff
@@ -25,15 +25,23 @@ val playDependencies = Def.setting[Seq[ModuleID]] {
     }
   }
 
+  val groupId = {
+    if (ver startsWith "2.") {
+      "com.typesafe.play"
+    } else {
+      "org.playframework"
+    }
+  }
+
   val baseDeps = base.map {
     case (name, scope) =>
-      ("com.typesafe.play" %% name % ver % scope) cross x
+      (groupId %% name % ver % scope) cross x
   }
 
   baseDeps
 }
 
-lazy val reactivemongo = Project("Play2-ReactiveMongo", file(".")).settings(
+lazy val reactivemongo = Project("Play-ReactiveMongo", file(".")).settings(
```

Comment:
If it update the artifactId, it cannot be done as-is to keep compat

Reply:
I have reverted the naming of the project, However i don't know how you see the publishing of the play3 version. Should it be another subproject ? another git project ? maybe a dynamic naming of the artifact for publishing?